Package.swift

```diff
@@ -29,7 +29,8 @@ import PackageDescription
 let package = Package(
     name: "PaperOnboarding",
     platforms: [
-        .iOS(.v10)
```

Comment:
Hey!) Good work, but let's keep .v10. Other things looks good 
Thank You 👍🏻

Reply:
can change, podspec says iOS version is 8 though, want to have swiftPM version different?

Reply:
yeah let's change, we updated version in Readme. And will update it in podspec later 
Hope nobody already not targeting to iOS 8 :)

Reply:
Makes sense, I'd be surprised if anyone was!
Just updated it now 👍